ARM-Samples/Create-Video-Analyzer/avam.template.json

```diff
@@ -0,0 +1,67 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+         "name": {
+            "type": "string",
+            "metadata" : {
+                "description" : "The name of the AVAM resource"
+            },
+            "defaultValue": "avam-account"
+        },
+        "classicAccountId": {
+            "type": "string",
+            "metadata" : {
+                "description" : "When migrating from Classic Account to Arm accout this value indicates what is the classic Account reource Id."
+            },
+            "defaultValue" :""
+        },
+        "managedIdentityId": {
+            "type": "string",
+            "metadata" : {
+                "description" : "The managed identity used to grant access to the Azure Media Service (AMS) account"
+            },
+            "defaultValue": "/subscriptions/<subscriptionId>/resourcegroups/<AVAMResourceGroup>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/<MIName>"
+        },
+        "mediaServicesAccountId": {
+            "type": "string",
+            "metadata" : {
+                "description" : "The media Service Account Id. The Account needs to be created prior to the creation of this template"

```

Comment:
I am not sure that account id is a good name, it is the account resource uri

Reply:
can you change it on the next Release ?

Reply:
the name in the resource schema is "resourceId" (under mediaServices") which is fine. it is just the name of the parameter that is accountId that I am not sure about, and that you can change freely if you agree

Reply:
done